lib/db_connection/connection_pool.ex

```diff
@@ -34,7 +34,8 @@ defmodule DBConnection.ConnectionPool do
       next: now_in_ms,
       poll: nil,
       idle_interval: idle_interval,
-      idle: nil
+      idle: nil,
+      connection_listeners: Keyword.get(opts, :connection_listeners) || []
```

Comment:
I assume we don't need this change anymore?

Reply:
Yup. I've just reverted this :)

Reply:
It seems that I really let this fly by me :sweat_smile: